sdk/monitor/azquery/models.go

```diff
@@ -224,28 +224,65 @@ type MetricNamespaceName struct {
 	MetricNamespaceName *string
 }
 
+// MetricResultsResponse - The metrics result for a resource.
+type MetricResultsResponse struct {
+	// The collection of metric data responses per resource, per metric.
+	Values []*MetricResultsResponseValuesItem
+}
+
+type MetricResultsResponseValuesItem struct {
```

Comment:
I think we should shorten this type name (I mentioned this on the APIView). I suggested `MetricValues` or `MetricData` but open to suggestions.

Reply:
I agree, it's a little too long. Changing it to MetricValues

Reply:
fixed in [d4ede52](https://github.com/Azure/azure-sdk-for-go/pull/21996/commits/d4ede52883798e7427f4a63972949b0568145ff5)